### PR TITLE
[Task][68358] Fetch Detail Data

### PR DESCRIPTION
**Pull request by TuanKieuDev**
### [Redmine link]
https://edu-redmine.sun-asterisk.vn/issues/68358?issue_count=6&issue_position=4&next_issue_id=68357&prev_issue_id=68361

#### Screenshots (nếu có)

https://github.com/awesome-academy/ios_intern_kieu_anh_tuan-b_project_1/assets/56763942/96dbfab1-18ff-46ef-8e1a-b78dd6132432

### CODE REVIEW

File: MarvelApp/MarvelApp/Controller/InformationDetail/InformationDetailViewController.swift [hunk 1]
```diff
@@ -36,19 +33,88 @@ final class InformationDetailViewController: UIViewController {
...
         detailTableView.delegate = self
         detailTableView.dataSource = self
     }
 
     @IBAction private func backButtonTapped(_ sender: Any) {
         self.navigationController?.popViewController(animated: true)
     }
+
+    private func configureData() {
+        if let description = overviewInformation?.description, description != "" {
```
Comment (hlinh88) [inline MarvelApp/MarvelApp/Controller/InformationDetail/InformationDetailViewController.swift:46]: define cho "" vi du nhu String.empty

File: MarvelApp/MarvelApp/Controller/InformationDetail/InformationDetailViewController.swift [hunk 2]
```diff
@@ -36,19 +33,88 @@ final class InformationDetailViewController: UIViewController {
...
+        }
+        if let events = overviewInformation?.events {
+            detailTableViewData.append(
+                CellData(
+                    opened: false,
+                    title: DetailSectionLabel.event.rawValue,
+                    sectionDetail: events.collectionURI ?? events.resourceURI ?? ""
+                )
+            )
+        }
```
Comment (hlinh88) [inline MarvelApp/MarvelApp/Controller/InformationDetail/InformationDetailViewController.swift]: tuong tu nhu tren

File: MarvelApp/MarvelApp/Controller/InformationDetail/InformationDetailViewController.swift [hunk 3]
```diff
@@ -59,7 +125,7 @@ extension InformationDetailViewController: UITableViewDataSource {
 
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         if section > DetailSection.image.rawValue && detailTableViewData[section - 1].opened {
-            return detailTableViewData[section - 1].sectionData.count + 1
+            return 2
         } else {
             return 1
```
Comment (hlinh88) [inline MarvelApp/MarvelApp/Controller/InformationDetail/InformationDetailViewController.swift]: nen define

File: MarvelApp/MarvelApp/View/Cell/MediaCollectionViewCell.swift [hunk 4]
```diff
@@ -0,0 +1,23 @@
+//
+//  MediaCollectionViewCell.swift
+//  MarvelApp
+//
+//  Created by Tobi on 22/09/2023.
+//
+
+import UIKit
+
+class MediaCollectionViewCell: UICollectionViewCell {
```
Comment (hlinh88) [inline MarvelApp/MarvelApp/View/Cell/MediaCollectionViewCell.swift]: final

**Review by hlinh88 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**